### PR TITLE
Firefox Android support

### PR DESCRIPTION
**Pull request by Sotilrac**
# Changes
* Minor js cleanup
* Updates deployment instructions
* Updates manifest for android compatibility.

### CODE REVIEW

**Comment by okhwaja** [comment]
Hey @Sotilrac , thanks for the PR. Checking now

File: manifest.json [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
+    "version": "1.3.0",
 
-  "description": "Sends the reader to the same article on https://outline.com",
-  "icons": {
-    "48": "icons/md-arrow-round-forward.svg",
-    "96": "icons/md-arrow-round-forward.svg"
-  },
+    "browser_specific_settings": {
+        "gecko": {
+            "id": "send-to@outline.com"
```
Comment (okhwaja) [inline manifest.json:9]: I feel a little uncomfortable using an email address I'm not associated with as the id for this extension

What do you think about updating this to a guid as described here? - https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Generating_GUIDs

File: background.js [hunk 2]
```diff
@@ -38,26 +38,18 @@ function findCurrentUrl(event) {
 
 function findOutlineUrl(currentUrl) {
   return Promise.resolve(`https://outline.com/${currentUrl}`)
-
```
Comment (okhwaja) [inline background.js:41]: 🙇

File: background.js [hunk 3]
```diff
@@ -38,26 +38,18 @@ function findCurrentUrl(event) {
...
-  id: "open-link-in-outline",
-  title: "Open Link in Outline",
+  id: "open-in-outline",
+  title: "Open in Outline",
   contexts: ["link"]
 });
 
 browser.contextMenus.onClicked.addListener((info, tab) => {
-  if (info.menuItemId === "open-link-in-outline") {
+  if (info.menuItemId === "open-in-outline") {
```
Comment (okhwaja) [inline background.js:52]: this change seems like a no-op. just curious - what prompted you to change the id?

**Review by okhwaja — COMMENTED** [review]
left a few comments with 1 suggested change. I'd be happy to merge once addressed. Thanks for submitting!

**Comment by okhwaja** [comment]
On second thought, i'll merge and update the guid. I won't have access to an android device till tomorrow to test...but I've confirmed it works on desktop so it's safe to publish. Thanks!

**Comment by Sotilrac** [comment]
Thanks a lot for changing to a guid. I wasn't sure what value to use since it could be an id of yours.

As for the "no-op" change in the menu id item. I changed it because it fits better on the android menu on my phone. Also, the word "link" could be removed and the meaning preserved.

Sorry for the missing newline. and thanks a lot for merging!

**Pull request merged**